### ISSUE TITLE
Create new index from sophos firewall logs as 'sophos' instead of using wazuh-alerts-...

### ISSUE THREAD

**Issue by tarekkilani** [wazuh/wazuh#10385]
Hello, 

Please i'm using ruleset and decoder to index sophos XG (firewall) to ELK. The logs are by default indexed in the "wazuh-alerts..." index. 
Is there any way to index the firewall logs in a dedicated index named "sophos"?
Thanks in advance for your help.

**Comment by flygoody** [comment]
I hope to get a solution. At the same time, I just used owlh. I only know that Suricata alerts in owlh are stored together with wazuh alerts, but I don't know how DNS and other alarm data generated by owlh are stored in a new index. So far, it has not been successfully established. It is generally configured in the configuration file of filebeat, but wazuh does not specify the exported index in filebeat, So very strange; In addition, thank the owlh developers very much@ 72nomada. I have learned a lot from your project. I hope the owlh project can get better and better

**Comment by flygoody** [comment]
I hope to get a solution. At the same time, I just used owlh. I only know that Suricata alerts in owlh are stored together with wazuh alerts, but I don't know how DNS and other alarm data generated by owlh are stored in a new index. So far, it has not been successfully established. It is generally configured in the configuration file of filebeat, but wazuh does not specify the exported index in filebeat, So very strange; In addition, thank the owlh developers very much    @72nomada . I have learned a lot from your project. I hope the owlh project can get better and better
